recipes/elfutils/all/conanfile.py

```diff
@@ -0,0 +1,185 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class ElfutilsConan(ConanFile):
+    name = "elfutils"
+    description = "A dwarf, dwfl and dwelf functions to read DWARF, find separate debuginfo, symbols and inspect process state."
+    homepage = "https://sourceware.org/elfutils"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "elfutils", "libelf", "libdw", "libasm")
+    exports = "patches/**"
+    license = ["GPL-1.0-or-later", "LGPL-3.0-or-later", "GPL-3.0-or-later"]
+    
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
This set of settings is useful for libraries but not for the tools. Should this package be broken in two?

Reply:
No, Conan received more cross build support recently.
When cross building (using this one as a build requirement), it should map the os/arch to os_build/arch_build.